apps/framework-cli/src/framework/python/executor.rs

```diff
@@ -42,6 +42,16 @@ pub static AGGREGATIONS_RUNNER: &str = include_str!("scripts/aggregations_runner
 pub static BLOCKS_RUNNER: &str = include_str!("scripts/blocks_runner.py");
 pub static CONSUMPTION_RUNNER: &str = include_str!("scripts/consumption_runner.py");
 
+const PYTHON_PATH: &str = "PYTHONPATH";
+fn python_path_with_version() -> String {
+    let mut paths = std::env::var(PYTHON_PATH).unwrap_or_else(|_| String::from(""));
+    if !paths.is_empty() {
+        paths.push(':');
+    }
+    paths.push_str(".moose/versions");
```

Comment:
will have to add it to `python.analysis.extraPaths` `python.autoComplete.extraPaths` in vs code probably